geoindicators/src/main/groovy/org/orbisgis/BuildingIndicators.groovy

```diff
@@ -76,3 +76,72 @@ static IProcess buildingSizeProperties() {
                 [outputTableName: outputTableName]
             }
     )}
+
+
+/**
+ * This process extract building interactions properties.
+ * @return A database table name.
+ * @author Jérémy Bernard
+ */
+static IProcess buildingNeighborsProperties() {
+    return processFactory.create(
+            "Building interactions properties",
+            [inputBuildingTableName: String,inputFields:String[],operations: String[]
+             , outputTableName: String, datasource: JdbcDataSource],
+            [outputTableName : String],
+            { inputBuildingTableName,inputFields, operations, outputTableName, datasource ->
+                def geometricField = "the_geom"
+                def idField = "id_build"
+                def height_wall = "height_wall"
+                def ops = ["building_contiguity","building_common_wall_fraction",
+                           "building_number_building_neighbor"]
+                // To avoid overwriting the output files of this step, a unique identifier is created
+                def uid_out = System.currentTimeMillis()
+                // Temporary table names
+                def build_intersec = "build_intersec"+uid_out.toString()
+
+
+                String query = "CREATE SPATIAL INDEX IF NOT EXISTS buff_ids ON $inputBuildingTableName($geometricField); " +
+                        "CREATE INDEX IF NOT EXISTS buff_id ON $inputBuildingTableName($idField);" +
```

Comment:
Please use
CREATE INDEX IF NOT EXISTS buff_ids ON $inputBuildingTableName($geometricField) USING rtree;

Reply:
I have tried to implement your piece of code for all my index creation but it does not work. I got the following error:
`Syntax error in SQL statement {0}; expected {1}; SQL statement:`